packages/flame_tiled/lib/src/tiled_component.dart

```diff
@@ -10,7 +11,7 @@ import 'package:flame_tiled/src/renderable_tile_map.dart';
 /// It uses a preloaded [RenderableTiledMap] to batch rendering calls into
 /// Sprite Batches.
 /// {@endtemplate}
-class TiledComponent extends Component {
+class TiledComponent extends Component with HasGameRef {
```

Comment:
Can you make it

```dart
class TiledComponent<T extends FlameGame> extends Component with HasGameRef<T> {
```

It will allow users to get their own gameRef of derived type.

Reply:
done